src/runner.js

```diff
@@ -132,19 +132,24 @@ assign(Runner.prototype, {
     return queryPromise
       .then((resp) => {
         const processedResponse = this.client.processResponse(resp, runner);
+        const postProcessedResponse = this.client
+          .postProcessResponse(processedResponse);
+
         this.builder.emit(
           'query-response',
-          processedResponse,
+          postProcessedResponse,
```

Comment:
Technically if a `postProcessResponse` function has been supplied in the config, this is no longer the `query-response`. But I'm not sure if it really matters given the context of the event. Just something I took note of.

Reply:
It hasn't been really query response earlier either, because each `knex` dialect is processing the raw response from driver before passing it to this event.